tests/README.md

```diff
@@ -0,0 +1,67 @@
+# Aeneas test suite
```

Comment:
Small nit: for the title, you should use `##` for the section title (i.e., `## Aeneas test suite`)`, and `###` for the subsections (e.g., `### Running the test suite`).

Reply:
Why? Is `#` too big?

Reply:
Oh, you might have misunderstood: this is not a new section in the main README, this is a new README file